src/infrastructure/ajv.ts

```diff
@@ -39,9 +37,11 @@ export function parseJsonOfSchema<T>(
 	schema: JSONSchemaTypeWithId<T>,
 ): T {
 	try {
-		const parsed: unknown = JSON.parse(ensureString(value));
-		assertSchema(parsed, schema);
-		return parsed;
+		if (typeof value === 'string') {
```

Comment:
**nipick:** Consider using  the existing `isString` function for a type assertion:

```ts
if (isString(value)) {
  value = JSON.parse(value);
}
```